registration-processor/post-processor/registration-processor-printing-stage/src/main/java/io/mosip/registration/processor/print/stage/PrintingStage.java

```diff
@@ -343,17 +355,20 @@ public MessageDTO process(MessageDTO object) {
 		return object;
 	}
 
-	private CredentialRequestDto getCredentialRequestDto(String regId) {
+	private CredentialRequestDto getCredentialRequestDto(String regId,String issuerId,String credentialType,String template) {
 		CredentialRequestDto credentialRequestDto = new CredentialRequestDto();
+		Map<String, Object> additionalAttributes=new HashMap<>();
 
-		credentialRequestDto.setCredentialType(env.getProperty("mosip.registration.processor.credentialtype"));
+		credentialRequestDto.setCredentialType(credentialType);
 		credentialRequestDto.setEncrypt(encrypt);
 
 		credentialRequestDto.setId(regId);
 
-		credentialRequestDto.setIssuer(env.getProperty("mosip.registration.processor.issuer"));
+		credentialRequestDto.setIssuer(env.getProperty(issuerId));
```

Comment:
I think the entire statement `env.getProperty("mosip.registration.processor.issuer")` needs to be replaced with the `issuerId` . We should not call env.getProperty with the issuerId.

Reply:
taken care